enhancements/results-forwarding.md

```diff
@@ -0,0 +1,379 @@
+---
+title: results-forwarding
+authors:
+  - TBD
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Compliance Results Forwarding / Storage
+
+## Summary
+
+This will allow for a flexible and extendible mechanism to forward compliance
+results and evidence to centralized stores, respectively.
+
+## Motivation
+
+While the current approach of generating results and remediations via CRDs
+has worked so far for Compliance Operator, this has several draw-backs that
+prove painful in multi-cluster environments:
+
+* `ComplianceCheckResults` and `ComplianceRemediations` exist in the cluster
+  that's the target for compliance checks, when expanding operations to multiple
+  clusters, this becomes cumbersome to check.
+
+* `ComplianceCheckResults` and `ComplianceRemediations` take space in Etcd. This
+  space may be better used by other operation-critical components in
+  resource-constrained deployments.
+  
+* By using CRDs we depend on Etcd, which is not a generic database
+  and doesn't provide all the assurances that one would expect:
+  
+  - It's non-trivial to do disaster recovery: An object snapshot contains
+    CRD metadata which is hard to replicate (e.g. object UUID) and is irrelevant
+    for compliance checks. Full etcd restore is a tedious and complicated process [1].
+
+  - Values are limited to a size of 1.5Mb which makes evidence storage
+    tricky and expensive. e.g. we currently use one PVC per `ComplianceScan`
+    for raw results which may be considered evidence.
+
+  - The current CRDs may be very verbose, which may result in big objects, which
+    may incur a penalty in performance and scalability for Kubernetes. [2]
+
+While projects such as StackRox [3] have successfully integrated Compliance Operator
+via CRDs, having a more lightweight approach 
+
+By sending results and evidences to a central store, we'll be able to:
```

Comment:
nit: s/be able to/enable/

Only as a distinction that this work is completely decoupled from whatever provides the benefits below.